build_files/20-install-apps.sh

```diff
@@ -6,11 +6,27 @@ dnf5 -y downgrade dnf5
 
 dnf5 install -y \
     android-tools \
+    bcc \
+    bpftop \
+    bpftrace \
+    flatpak-builder \
+    nicstat \
+    numactl \
     openrgb \
+    p7zip-plugins \
+    podman-machine \
+    podman-tui \
+    python3-ramalama \ 
```

Comment:
```suggestion
    python3-ramalama \
```
This was breaking the build